src/main/java/picard/analysis/CollectQualityYieldMetrics.java

```diff
@@ -50,6 +50,8 @@
         programGroup = Metrics.class
 )
 public class CollectQualityYieldMetrics extends SinglePassSamProgram {
+    private QualityYieldMetricsCollector collector = new QualityYieldMetricsCollector();
```

Comment:
I would like the defaults to be stored in one place, so that if you call `CollectQualityYieldMetrics` with no command line options specified, or instantiate a `QualityYieldMetricsCollector`, you get the same defaults.  The other option is to have `static final` variables in `QualityYieldMetricsCollector` that can be referenced by `CollectQualityYieldMetrics`.  Does this make sense what I am trying to achieve?